src/tests/test_collection.py

```diff
@@ -101,14 +101,33 @@ def test_delete(client: vecs.Client) -> None:
     # insert works
     movies.upsert(records)
 
+    # delete by IDs.
     delete_ids = ["vec0", "vec15", "vec99"]
     movies.delete(ids=delete_ids)
     assert len(movies) == n_records - len(delete_ids)
 
+    # insert works
+    movies.upsert(records)
+
+    # delete with filters
+    genre_to_delete = "action"
+    deleted_ids_by_genre = movies.delete(
+        filters={"genre": {"$eq": genre_to_delete}})
+    assert len(deleted_ids_by_genre) > 0
+
     # bad input
     with pytest.raises(vecs.exc.ArgError):
         movies.delete(ids="should_be_a_list")
 
+    # bad input: neither ids nor filters provided.
+    with pytest.raises(vecs.exc.ArgError):
+        movies.delete()
+
+    # bad input: should only provide either ids or filters, not both
+    with pytest.raises(vecs.exc.ArgError):
```

Comment:
@olirice I'm a bit stumped on why this test fails. Any ideas? If both ids and filters are not None, I raise an error in the delete function. Any idea if I am missing something here?

Reply:
Seems to be passing now. Must have been a cache issue.